src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [filmName, setFilmName] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+
+    const movieInfo = `${title} ${description}`.toLowerCase();
+    const normalizedFilmName = filmName
```

Comment:
super low-level logic here, don't think this belongs to the main component logic. I would move this one to a separate `helpers.ts` file